app/forms/hyrax/publication_form.rb

```diff
@@ -1,10 +1,18 @@
 # Generated via
 #  `rails generate hyrax:work Publication`
+require 'rdf/markdown'
+
 module Hyrax
   class PublicationForm < Hyrax::Forms::WorkForm
     self.model_class = ::Publication
     self.terms += [:resource_type]
     self.terms += ::Attributes.to_a
     self.required_fields = [:title]
+
+    def self.model_attributes(form_params)
+      result = super
+      result[:related_url].map! { |related_url| RDF::Markdown::Literal.new(related_url) }
```

Comment:
Does this work when editing an existing related URL field?  (When you are editing the work vs creating a new work)

Reply:
To answer my own question:  Yes, it's working just fine for editing too